src/pdm/cli/templates/default/pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[project]
+name = "example-package"
+version = "0.1.0"
+description = "Default template for PDM package"
+authors = []
+dependencies = []
+requires-python = ">=3.7"
```

Comment:
I guess you could use 3.8 since 3.7 is EOL in two days.

Reply:
Updated. This value is a placeholder and will be replaced by what the users input.